sklearn/plot/_gridsearch_results.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
+from sklearn.plot import plot_heatmap
+
+
+def plot_gridsearch_results(cv_results, param_grid, metric='mean_test_score',
```

Comment:
This only works for 2d grid-searches, right? I think we should also support 1d, and error for anything else.